src/ruby/meetings/Gemfile

```diff
@@ -58,14 +58,12 @@ gem 'draper', github: 'drapergem/draper'
 gem 'haml-rails'
 gem 'country_select'
 gem 'kaminari'
+gem "htmlentities"
 gem 'factory_girl_rails'
 gem 'active_model_serializers', '~> 0.10.0'
 gem 'activemodel-serializers-xml'
 gem 'swagger-blocks'
 gem 'jsonapi-resources'
-gem 'globalize', git: 'git@github.com:JohnSmall/globalize.git'
-gem 'rack-cors', :require => 'rack/cors'
```

Comment:
This is also a regression. Since you're not currently around, I think I'll just copy your PR branch, fix the issues, and submit a new PR based on it.